docs/assets/scss/_elements.scss

```
@@ -139,7 +148,14 @@ pre,
 code,
 kbd,
 samp {
-  font-family: SFMono-Regular, Menlo, Monaco, Consolas, liberation mono, courier new, monospace;
+  font-family:
+    SFMono-Regular,
+    Menlo,
+    Monaco,
+    Consolas,
+    liberation mono,
```

Stylelint (17.14.0)

[error] Expected quotes around "liberation mono" (font-family-name-quotes)

(font-family-name-quotes)